httpbp/prometheus.go

```diff
@@ -0,0 +1,115 @@
+package httpbp
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+
+	"github.com/reddit/baseplate.go/prometheusbp"
+)
+
+const (
+	methodLabel            = "http_method"
+	successLabel           = "http_success"
+	endpointLabel          = "http_endpoint"
+	codeLabel              = "http_response_code"
+	remoteServiceSlugLabel = "http_slug"
+)
+
+var (
+	serverLabels = []string{
+		methodLabel,
+		successLabel,
+		endpointLabel,
+	}
+
+	serverLatency = promauto.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    "http_server_latency_seconds",
+		Help:    "RPC latencies",
```

Comment:
nit: I would say HTTP endpoints are not RPCs but just that: HTTP endpoint latencies or just Endpoint latencies.

Reply:
oops, copy/paste snafu